Algorithms/Sequences/PrimesSequence.cs

```diff
@@ -20,27 +20,26 @@ public class PrimesSequence : ISequence
         /// <summary>
         /// Gets sequence of prime numbers.
         /// </summary>
-        public IEnumerable<BigInteger> Sequence
+        public IEnumerable<BigInteger> Sequence => GeneratePrimeSequence();
```

Comment:
The same here, please rollback.

Reply:
@siriak Sequences are rolled back to previous implementations